### PR TITLE
Refactor planar sse

### PR DESCRIPTION
**Pull request by peppsac**
Planar mode sse used a simplistic computation until now.

This PR proposes to change it to a visibilty based one: we subdivide based one the on-screen size of each tile.

WDYT?

### CODE REVIEW

File: src/Process/PlanarTileProcessing.js [hunk 1]
```diff
@@ -9,47 +8,24 @@ export function planarCulling(node, camera) {
...
+    const dim = {
+        x: 0.5 * (onScreen.max.x - onScreen.min.x),
+        y: 0.5 * (onScreen.max.y - onScreen.min.y),
+    };
 
-    if (ratio >= 0.25) {
-        return 7;
-    }
-    return 1;
+    return (dim.x > 0.3 && dim.y >= 0.3);
```
Comment (autra) [inline src/Process/PlanarTileProcessing.js]: Can you elaborate on the magic constants (0.3 here and the multiplication by 0.5 above) and about the `>` for x and `>=` for y?

`0.3` is the equivalent of the old `threshold`, isn't it?

File: src/Core/Prefab/PlanarView.js [hunk 2]
```diff
@@ -101,7 +100,7 @@ function PlanarView(viewerDiv, boundingbox, options) {
     tileLayer.update =
         processTiledGeometryNode(
             planarCulling,
-            planarSubdivisionControl(4, SSE_SUBDIVISION_THRESHOLD),
+            planarSubdivisionControl(options.maxSubdivisionLevel || 5),
```
Comment (autra) [inline src/Core/Prefab/PlanarView.js:103]: Maybe obvious question, but why did you change the default maxLevel from 4 to 5?

**Review by autra — COMMENTED** [review]
Nice! It feels more natural and better to adopt a perspective approach to this problem. I just have a few questions.

File: src/Process/PlanarTileProcessing.js (hunk 1 above)
Comment (peppsac) [reply src/Process/PlanarTileProcessing.js]: `0.5` is needed because `box3DSizeOnScreen` result goes from `-1` to `+1` (for each axis), so we get back to `[0, 1]`.

The `>` vs `>=` is a typo, I'll fix this.

The `0.3` value are `threshold` that looks good on the sole example of planar mode that we have.
I can make them configurable if you want but honestly this PR is a first iteration and we will probably need to adjust it later (for instance to depend on the resolution of the images we get from the layer).

File: src/Core/Prefab/PlanarView.js (hunk 2 above)
Comment (peppsac) [reply src/Core/Prefab/PlanarView.js:103]: `4`  or `5` are both hand-picked values that look good.
Again, we probably want something smarter (e.g depending on the area covered?) but that's for a another PR...

**Review by autra — APPROVED** [review]
LGTM
